app/src/main/java/de/danoeh/antennapod/activity/MainActivity.java

```diff
@@ -657,6 +658,15 @@ public boolean onKeyUp(int keyCode, KeyEvent event) {
                     return true;
                 }
                 break;
+            case KeyEvent.KEYCODE_B:
+            case KeyEvent.KEYCODE_T:
+                OnKeyListenerForFragments myFragment =
+                        (OnKeyListenerForFragments) getSupportFragmentManager()
+                                .findFragmentById(R.id.main_view);
```

Comment:
This looks like it will crash when pressing the keys for any fragment that is not the feed item list. Instead, I would take the event and send it using `EventBus.getDefault().post(event)`. You can then `@Subscribe` to it in the fragment directly (see other methods with that annotation in the fragment). This also reduces coupling between fragments and main activity.

Reply:
Thank you for suggestion. I have changed my code like you said. I will be waiting.